lib/agent.js

```diff
@@ -324,6 +327,7 @@ Agent.prototype.captureError = function (err, opts, cb) {
 
     if (agent._transport) {
       agent.logger.info(`Sending error to Elastic APM`, { id: error.id })
+      agent._metrics.getOrCreateCounter('agent.errors.sent').inc()
```

Comment:
We decided to delay the introduction of `agent.*` metrics, so this line should just be removed